### ISSUE TITLE
Training curve drops sharply and then plateaus prematurely during fine-tuning, failing to match pre-trained performance

### ISSUE THREAD

**Issue by XElcine** [robodhruv/visualnav-transformer#70]
Hello, I am encountering an issue when retraining the  model nomad   

I observe that the training loss drops very sharply within several epoches  but then immediately plateaus

Ultimately, the model's performance on validation set (and train set ) is  worse than the performance reported in the repository for the pre-trained model
I have chaged the datasets and environments but the results stay the same
>

**Comment by XElcine** [comment]
<img width="2263" height="1189" alt="Image" src="https://github.com/user-attachments/assets/c149b3fc-8885-4100-b11c-94ae035867b9" />
